examples/name_registry/name_registry.vy

```diff
@@ -1,3 +1,4 @@
+#pragma version >0.3.10
```

Comment:
do we need the pragma statements in this codebase?

every test that uses these automatically is compiled with that specific version, so I don't think we need to deal with the maintenance burden of having to continuously update these every release

Reply:
the `@deploy` decorator won't compile before this version

Reply:
i don't think we need to update the pragmas every release, only on breaking language changes